cf/test/test_Query.py

```diff
@@ -47,6 +50,13 @@ def test_Query(self):
         str(u)
         u.dump(display=False)
 
+        # For "wi", check repr. provides correct notation for open/closed-ness
+        # of the interval captured.
+        self.assertEqual(repr(q), "<CF Query: (wi [2, 5])>")
+        self.assertEqual(repr(v), "<CF Query: (wi (2, 5])>")
+        self.assertEqual(repr(w), "<CF Query: (wi [2, 5))>")
+        self.assertEqual(repr(x), "<CF Query: (wi (2, 5))>")
```

Comment:
When updating the PR I was thinking I'd consider it an improvement if we add a space after the stated interval, to make this more readable and highlight the bracket/parenthesis notation, e.g. instead of `<CF Query: (wi [2, 5])>` we would have `<CF Query: (wi [2, 5] )>` (or maybe `<CF Query: ( wi [2, 5] )>` for symmetry?) and so on.

Not sure if you agree? Even if you do, I don't mind leaving it out of this PR though, since the release is imminent and that is a new commit I'd need to update the PR with.